### PR TITLE
cgminer: update to 4.11.1.

### PR DESCRIPTION
**Pull request by CameronNemo**

### CODE REVIEW

File: srcpkgs/cgminer/template [hunk 1]
```diff
@@ -15,7 +15,7 @@ maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/cgminer/template]: please use SPDX short-identifier

File: srcpkgs/cgminer/template [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'cgminer'
 pkgname=cgminer
-version=4.10.0
+version=4.11.1
 revision=1
 build_style=gnu-configure
 configure_args="
```
Comment (fosslinux) [inline srcpkgs/cgminer/template]: Why is the next line not on this one (or at least some of the configure options)?

File: srcpkgs/cgminer/template [hunk 3]
```diff
@@ -12,10 +12,10 @@ makedepends="libcurl-devel libusb-devel jansson-devel eudev-libudev-devel ncurse
...
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-license="GPL-3"
+license="GPL-3.0-or-later"
 homepage="https://github.com/ckolivas/cgminer"
 distfiles="http://ck.kolivas.org/apps/${pkgname}/${pkgname}-${version}.tar.bz2"
-checksum=debd49291f8335b48496896dc28a43f941d1c981525950497acc89d0d5fc1e84
+checksum=a8f3e244343eda8c3410e6d27188ee941de25c54bd76412e5b4d17de9f30ac18
 
 pre_configure() {
 	# We have latest jansson and libusb - just use them
```
Comment (fosslinux) [inline srcpkgs/cgminer/template]: The following seds - why so much space between the sed and the options?

File: srcpkgs/cgminer/template (hunk 2 above)
Comment (Vaelatern) [reply srcpkgs/cgminer/template]: Accepted pattern for long lines.

File: srcpkgs/cgminer/template (hunk 3 above)
Comment (Vaelatern) [reply srcpkgs/cgminer/template]: Probably to line up to tabs.

**Pull request merged**